pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -250,7 +250,7 @@ func (p *PriorityQueue) Add(pod *v1.Pod) error {
 	defer p.lock.Unlock()
 	pInfo := p.newQueuedPodInfo(pod)
 	if err := p.activeQ.Add(pInfo); err != nil {
-		klog.Errorf("Error adding pod %v to the scheduling queue: %v", nsNameForPod(pod), err)
+		klog.ErrorS(err, "Error adding pod to the scheduling queue", klog.KObj(pod))
```

Comment:
Not follow the K/V pattern

Reply:
@gavinfish what do you mean by that?

Reply:
It should be like 

klog.ErrorS(err, "Error adding pod to the scheduling queue", `"pod"`, klog.KObj(pod))